docs/console.rst

```diff
@@ -0,0 +1,149 @@
+Console
+=======
+
+.. autoclass:: ev3dev2.console.Console
+    :members:
+
+Examples:
+
+.. code-block:: py
+
+    #!/usr/bin/env micropython
+    from ev3dev2.console import Console
+
+    # create a Console instance, which uses the default font
+    console = Console()
+
+    # reset the console to clear it, home the cursor at 1,1, and then turn off the cursor
+    console.reset_console()
+
+    # display 'Hello World!' at row 5, column 1 in inverse, but reset the EV3 LCD console first
+    console.text_at('Hello World!', column=1, row=5, reset_console=True, inverse=True)
+
+.. code-block:: py
+
+    #!/usr/bin/env micropython
+    from time import sleep
+    from ev3dev2._platform.ev3 import INPUT_1, INPUT_2, INPUT_3
```

Comment:
Here you want
```
from ev3dev2.sensor import INPUT_1, INPUT_2, INPUT_3
```
so that it will work on any platform

Reply:
Done!